internal/pkg/scaffold/cmd_test.go

```diff
@@ -111,6 +113,11 @@ func main() {
 
 	ctx := context.TODO()
 
+	// Start serving operator specific metrics.
+	if err := operatormetrics.ServeOperatorSpecificMetrics(cfg, metricsHost, operatorMetricsPort); err != nil {
+		log.Error(err, "")
+	}
```

Comment:
This function only exists after `operator-sdk add api` is called, so this would introduce compilation errors to a newly scaffolded project.